app/src/main/res/layout/fragment_help_question.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingStart="@dimen/fragment_padding"
+            android:paddingEnd="@dimen/fragment_padding"
+            android:paddingBottom="@dimen/fragment_padding_bottom_larger">
+
+            <TextView
+                android:id="@+id/question"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/Erouska.Body"
+                tools:text="Jak eRouška zaznamenává a zpracovává data o setkáních uživatelů?" />
+
+            <TextView
+                android:id="@+id/answer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/Erouska.Caption"
+                android:autoLink="web"
```

Comment:
let's use at least` android:autoLink="all"` or at least `android:autoLink="web|email"`

Reply:
![Uploading device-2021-01-21-120742.png…]()

Reply:
this is not needed, removed